.codespellignore

```diff
@@ -6,3 +6,5 @@ leapyears
 leapYears
 falsy
 warmup
+procs
+enumerables
```

Comment:
The Because and Why? sections of the PR don't explain what this change is doing. Can you elaborate on this?